pkg/apis/app/v2/kieapp_types.go

```diff
@@ -152,18 +152,32 @@ type KieServerSet struct {
 	KafkaJbpmEventEmitters *KafkaJBPMEventEmittersObject `json:"kafkaJbpmEventEmitters,omitempty"`
 	Cors                   *CORSFiltersObject            `json:"cors,omitempty"`
 	// MDBMaxSession number of KIE Executor sessions
-	MDBMaxSession *int `json:"MDBMaxSession,omitempty"`
+	MDBMaxSession    *int              `json:"MDBMaxSession,omitempty"`
+	TerminationRoute TerminationObject `json:"termination,omitempty"`
+}
+
+type TerminationObject struct {
+	// If set to true disableSSL will be skipped for the console the target port will be http and the termination will be edge
+	EnableInsecureEdgeTermination bool `json:"enableInsecureEdgeTermination,omitempty"`
+	// +kubebuilder:validation:Enum:=http;https
+	// Protocol to use, if no value is provided, https is the default protocol, if the EnableInsecureEdgeTermination is true http will be used
+	Protocol string `json:"protocol,omitempty"`
```

Comment:
do we really need to have there protocol property? In the requirement is specified only http. Also I would expect some more configuration may be needed for https.